scripts/cli/test.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -e
+npm run build
```

Comment:
I had failing tests and didn't realize I had to rebuild them.

This runs quickly and stops old versions from running.